manager/eks_cluster.yaml

```diff
@@ -19,3 +19,5 @@ metadata:
   name: $CORTEX_CLUSTER_NAME
   region: $CORTEX_REGION
   version: "1.14"
+
+availabilityZones: $CORTEX_AVAILABILITY_ZONES
```

Comment:
Should we add this in the nodegroup eks yaml files as well? According to their [docs](https://eksctl.io/usage/schema/), there is also a `availabilityZones` config under `NodeGroup`, should we add it there in each file as well?